draft-ietf-tls-tls13.md

```diff
@@ -2183,8 +2183,10 @@ hidden by TCP.
 
 A small allowance for errors in clocks and variations in measurements
 is advisable.  However, any allowance also increases the opportunity
-for replay.  In this case, it is better to reject early data than to
-risk greater exposure to replay attacks.
+for replay.  In this case, it is better to reject early data and fall back
+to a full 1-RTT handshake than to risk greater exposure to replay attacks,
+so in common network topologies, a small allowance on the order of ten
+seconds is reasonable.
```

Comment:
Among mobile clients resuming with a ticket that was issued within the past day, we see about .1% outside a window of +/- 30s, .4% outside +/- 10s, and 2% outside +/- 2s (this is among clients that can successfully validate our certificate).

That said, I don't think it makes sense to set a very small time limit without also preventing clients from slow trickling data in after the client hello. Setting a very small limit as is will prevent legitimate clients from doing 0-RTT while leaving a loophole open for malicious clients to extend the replay window. "Tens of seconds" sounds like reasonable guidance to me.